sencha-workspace/SlateDemonstrationsStudent/app/controller/Viewport.js

```diff
@@ -11,22 +11,25 @@ Ext.define('SlateDemonstrationsStudent.controller.Viewport', {
     
     // controller configuration
     views: [
-        'Dashboard@Slate.cbl.view.student',
-        'RecentProgress@Slate.cbl.view.student'
+        'CompetencyCard',
+        'Dashboard',
+        'DashboardController',
```

Comment:
@aliyeysides `DashboardController` shouldn't exist anymore, and it definitely shouldn't be getting loaded by an application controller's views list. A ViewController should get loaded by the view that's bound to it (so in this case your controller requires `Dashboard` via views which in-turn requires its controller) -- but in this case it should be gone entirely